lib/openhab/dsl/items/string_item.rb

```diff
@@ -15,13 +15,13 @@ class StringItem
         extend Forwardable
 
         include Comparable
-        include OpenHAB::DSL::Items::ItemDelegate
+        extend OpenHAB::DSL::Items::ItemDelegate
 
         # @return [Regex] Regular expression matching blank strings
         BLANK_RE = /\A[[:space:]]*\z/.freeze
         private_constant :BLANK_RE
 
-        def_delegator :@string_item, :to_s
+        def_item_delegator :@string_item, :to_s
```

Comment:
Why not use the default which includes groups?

Reply:
Because I missed that one.  :)